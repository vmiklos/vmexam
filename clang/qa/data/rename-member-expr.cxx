class C
{
public:
    int nX;
};

int main()
{
    C aC;
    aC.nX = 1;
    int y = aC.nX;
}
