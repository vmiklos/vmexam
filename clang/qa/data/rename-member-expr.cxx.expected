class C
{
public:
    int m_nX;
};

int main()
{
    C aC;
    aC.m_nX = 1;
    int y = aC.m_nX;
}
